src/Shouldly.Shared/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs

```diff
@@ -147,6 +147,24 @@ public static void ShouldNotBeEmpty<T>(this IEnumerable<T> actual, [InstantHandl
                 throw new ShouldAssertException(new ExpectedShouldlyMessage(actual, customMessage).ToString());
         }
 
+        public static T ShouldHaveSingleItem<T>(this IEnumerable<T> actual)
+        {
+            return ShouldHaveSingleItem(actual, () => null);
+        }
+
+        public static T ShouldHaveSingleItem<T>(this IEnumerable<T> actual, string customMessage)
+        {
+            return ShouldHaveSingleItem(actual, () => customMessage);
+        }
+
+        public static T ShouldHaveSingleItem<T>(this IEnumerable<T> actual, [InstantHandle] Func<string> customMessage)
+        {
+            if (actual == null || (actual != null && actual.Count() != 1))
```

Comment:
That should suffice. I did see that same pattern in these Enumerable Extension Methods as well:

```cs
        public static void ShouldNotBeEmpty<T>(this IEnumerable<T> actual, [InstantHandle] Func<string> customMessage)
        {
            if (actual == null || actual != null && !actual.Any())
                throw new ShouldAssertException(new ExpectedShouldlyMessage(actual, customMessage).ToString());
        }
```
And
```cs
        public static void ShouldBeEmpty<T>(this IEnumerable<T> actual, [InstantHandle] Func<string> customMessage)
        {
            if (actual == null || (actual != null && actual.Count() != 0))
                throw new ShouldAssertException(new ExpectedShouldlyMessage(actual, customMessage).ToString());
        }
```

Should I update these as well in this PR? Also seems like we could use ```Any()``` in ```ShouldBeEmpty()``` as well for consistency.